src/main/xar-resources/data/xquery/xquery.xml

```diff
@@ -67,17 +67,13 @@
                     xlink:href="https://www.w3.org/TR/xpath-functions-31/#func-xml-to-json">
                     fn:xml-to-json()
                   </link>
-                    implements handling of xml null, xml boolean, xml number, xml array and xml
-                    map elements according to specification.
-                    fn:xml-to-json() implements handling of xml strings and xml map keys that are not
-                    mentioned below according to specification. It ignores the options map.
-                    fn:xml-to-json() is missing implementation for replacing codepoints in the range
-                    1-31 or 127-159 by escaped unicode representations of the codepoint value for any
-                    xml string or map key.
-                    In addition to that for xml strings and map keys that are marked as beeing escaped
-                    fn:xml-to-json() is missing implementation to replace any unescaped occurrence of
-                    quotation mark, backspace, form-feed, newline, carriage return, tab, or solidus
-                    by \&quot;, \b, \f, \n, \r, \t, or \/ respectively.
+                    handles most valid input according to specification. Notable exceptions follow.
+                    <literal>fn:xml-to-json()</literal> does not replace codepoints in the range
+                    <literal>1-31</literal> or <literal>127-159</literal> in any
+                    string or map key.
+                    Additionally for strings and map keys that are marked as beeing escaped
```

Comment:
much better, thank you. One last thing: what does `marked as being escaped` mean? Something different from `escaped` and if so how? There is a typo in ~~beeing~~

Reply:
The function takes xml input and produces json output. we must ensure it produces valid json. so there must not be unescaped input in the output.
the spec says some input elements can have an attribute in their xml tag claiming their text is already a valid json string (with special characters escaped). this leads to different treatment of the input. the claim might also be a lie. so there is a difference between 'escaped' and 'marked as being escaped'.

Reply:
The non supported options `escaped="true"` and `escaped-key="true"` need to be made explicit in the text (inside a `<literal>`).  Since we're dealing with different realms of escaping in `xml` and `json` i think the best way to clarify this is by using [listing](https://www.exist-db.org/exist/apps/doc/author-reference#block).
```xquery
(: not supported :)
fn:xml-to-json(<string escape="true">string with  \b escaped codepoint</string>)

(: supported :)
fn:xml-to-json(<map key='cities'>
      <array key="Brussels">
        <map>
            <string key="to">London</string>
            <number key="distance">322</number>
        </map> 
        <map>
            <string key="to">Paris</string>
            <number key="distance">265</number>
        </map>
      </array>
      <array key="London">
        <map>
            <string key="to">Brussels</string>
            <number key="distance">322</number>
        </map> 
        <map>
            <string key="to">Paris</string>
            <number key="distance">344</number>
        </map> 
      </array>
    </map> ) 
```